### ISSUE TITLE
Remove alt text from icons

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1651]
Our icons are decorative elements and should not have alt text. The icons do not convey anything not stated in text and so adding alt text adds audio clutter. I added a clarifying flag to the components page: 
<img width="263" alt="image" src="https://github.com/cal-itp/benefits/assets/5394204/32705a70-0e99-4abf-b85c-e3d85cd051b6">

_Originally posted by @srhhnry in https://github.com/cal-itp/benefits/issues/1410#issuecomment-1678135953_

## Acceptance criteria

- [x] All of the icons in the image above have `alt=""` for their alt text attribute
- [x] All prior used alt text is removed from the PO files
- [x] Copy Master spreadsheet does not have alt text strings for icons.
